origin-growth/src/enums.js

```diff
@@ -0,0 +1,52 @@
+class Enum extends Array {
+  constructor(...args) {
+    super(...args)
+
+    for(const k of args) {
+      this[k] = k
+    }
+  }
+}
+
+const GrowthEventStatuses = new Enum(
+  'Logged',
+  'Verified',
+  'Fraud'
+)
+
+const GrowthEventTypes = new Enum(
+  'ProfilePublished',
+  'EmailAttestation',
+  'FacebookAttestation',
+  'AirbnbAttestation',
+  'TwitterAttestation',
+  'PhoneAttestation',
+  'ListingCreated',
+  'ListingPurchased'
+)
+
+const GrowthRewardStatuses = new Enum(
+  'Pending',
+  'Fraud,',
+  'Blocked',
+  'Paid'
+)
+
+const GrowthInviteContactTypes = new Enum(
+  'Email',
+  'Phone'
```

Comment:
I think this one should also have `Other` invite type.

Reply:
oh yes. I forgot. added. thank you !